src/dexie-storage.ts

```diff
@@ -1,18 +1,19 @@
 import Dexie from "dexie";
-import { OfflineActivity, IExportableAnswerMetadata } from "./types";
+import { OfflineActivity } from "./types";
+import { IIndexedDBAnswer } from "./storage-facade";
 
 // Copy and pasted from the example: https://dexie.org/docs/Typescript
 export class DexieStorage extends Dexie {
   // Declare implicit table properties.
   // (just to inform Typescript. Instantiated by Dexie in stores() method)
-  answers: Dexie.Table<IExportableAnswerMetadata, string>; // number = type of the primkey
   offlineActivities: Dexie.Table<OfflineActivity, string>;
+  answers: Dexie.Table<IIndexedDBAnswer, string>; // number = type of the primkey
   //...other tables goes here...
 
   constructor () {
       super("ActivityPlayer");
-      this.version(2).stores({
-          answers: "id, question_id, question_type",
+      this.version(4).stores({
```

Comment:
Seems weird that version 4 comes before version 3.  My reading of the docs was that you added versions and left old versions alone.  New versions can add columns to existing tables as shown in the docs (https://dexie.org/docs/Tutorial/Design#database-versioning):

```
In Dexie >= 3.0:

Just increase the version number and add shoeSize along the indexes:
db.version(2).stores({friends: "++id,name,shoeSize"});
```

Reply:
Version 3 was local, it just wasn't committed before being surpassed by 4